src/main/java/com/st/utopia/cancellationservice/model/Ticket.java

```diff
@@ -0,0 +1,103 @@
+package com.st.utopia.cancellationservice.model;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * An airport model.
+ *
+ * @author Henry Cho
+ * 
+ * Model used for the purpose of the cancellation microservice.
+ * 
+ * There are no set methods for information about the ticket such as flight number, row/seat, or class.
+ * 
+ * Upon cancelling the ticket, the microservice needs to set the following fields as null:
+ * 	-reserver
+ * 	-price
+ * 	-reservation_timeout
+ * 	-booking_id
+ * 
+ * This model does not need to define a composite key since the cancellation microservice will search for the ticket via booking_id which is unique.
+ */ 
+
+@Entity
+@Table(name="tbl_tickets")
+public class Ticket{
+	@Column(name="flight")
+	Integer flight;
+	@Column(name="row")
+	Integer row;
+	@Column(name="seat")
+	Character seat;
+	@Column(name="class")
+	Integer tier; //class is a keyword =(
+	@Column(name="reserver")
+	Integer reserver;
+	@Column(name="price")
+	Integer price;
+	@Column(name="reservation_timeout")
+	LocalDateTime reservation_timeout;
```

Comment:
Because JPA will translate camel-case to underscore-separated words, name fields using standard Java conventions (this should be `reservationTimeout`).